artman/pipelines/grpc_generation.py

```diff
@@ -106,19 +111,23 @@ class _GoGrpcTaskFactory(GrpcTaskFactoryBase):
     """
 
     def _get_grpc_codegen_tasks(self, **kwargs):
-        return [protoc_tasks.ProtoAndGrpcCodeGenTask,
-                protoc_tasks.GoExtractImportBaseTask,
-                protoc_tasks.GoLangUpdateImportsTask]
+        return [
+            protoc_tasks.ProtoAndGrpcCodeGenTask,
+            protoc_tasks.GoExtractImportBaseTask,
+            protoc_tasks.GoLangUpdateImportsTask,
+        ]
 
     def get_validate_kwargs(self):
-        return ['gapic_api_yaml', 'final_repo_dir'] + code_gen.COMMON_REQUIRED
+        return ['gapic_api_yaml', 'gapic_code_dir'] + code_gen.COMMON_REQUIRED
```

Comment:
@ethanbao I looked at this more. I think that `output_dir` and `gapic_code_dir` should be refactored, but I would prefer do this in a follow-up PR if that is okay with you. (The _status quo_ represents only a trivial change from the _status quo ante_.)